src/controllers/aer_controller.hpp

```diff
@@ -1738,7 +1748,9 @@ Controller::simulation_methods(std::vector<Circuit> &circuits,
         kraus_enabled = true;
       }
     }
-    return sim_methods;
+    if (!sim_methods.empty()) {
```

Comment:
I don't think this is necessary since sim methods will never be empty unless there are no circuits.